components/layout/navbar/index.tsx

```diff
@@ -4,28 +4,29 @@ import { Suspense } from 'react';
 import Cart from 'components/cart';
 import CartIcon from 'components/icons/cart';
 import LogoIcon from 'components/icons/logo';
-import { getMenu } from 'lib/shopify';
-import { Menu } from 'lib/shopify/types';
+import { getMenu } from 'lib/bigcommerce';
+import { VercelMenu as Menu } from 'lib/bigcommerce/types';
 import MobileMenu from './mobile-menu';
 import Search from './search';
 
 export default async function Navbar() {
   const menu = await getMenu('next-js-frontend-header-menu');
+  const demoMenu = menu.slice(0, 4);
```

Comment:
🍹 Let's throw this logic inside `getMenu`, that way our view logic doesn't need to know about any "business" logic.